src/blueprints/tv2/action-factories/tv2-camera-action-factory.ts

```diff
@@ -28,7 +28,7 @@ export class Tv2CameraActionFactory {
   ) {}
 
   public createCameraActions(blueprintConfiguration: Tv2BlueprintConfiguration): Action[] {
-    return blueprintConfiguration.studio.SourcesCam
+    return blueprintConfiguration.studio.cameraeSources
```

Comment:
typo in the rename.
`cameraeSources` -> `cameraSources`

Reply:
Fixed